hcache/kc.c

```diff
@@ -67,15 +67,18 @@ static void *hcache_kyotocabinet_open(const char *path)
 /**
  * hcache_kyotocabinet_fetch - Implements HcacheOps::fetch()
  */
-static void *hcache_kyotocabinet_fetch(void *ctx, const char *key, size_t keylen)
+static void *hcache_kyotocabinet_fetch(void *ctx, const char *key, size_t keylen, size_t *dlen)
 {
   size_t sp;
+  void *rv;
 
   if (!ctx)
     return NULL;
 
   KCDB *db = ctx;
-  return kcdbget(db, key, keylen, &sp);
+  rv = kcdbget(db, key, keylen, &sp);
```

Comment:
You can get rid of the `sp` variable and directly pass `dlen`

Reply:
Done